File: jawa/venator/src/main/java/ru/prohor/universe/venator/webhook/model/ApiResponse.java
package ru.prohor.universe.venator.webhook.model;

public record ApiResponse(
        boolean success,
        String message
) {
    public static ApiResponse success(String message) {
        return new ApiResponse(true, message);
    }

    public static ApiResponse error(String message) {
        return new ApiResponse(false, message);
    }
}

File: jawa/venator/src/main/kotlin/webhook/model/ApiResponse.kt
package ru.prohor.universe.venator.webhook.model

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
data class ApiResponse(
    val success: Boolean,
    val message: String
) {
    companion object {
        fun success(message: String): ApiResponse {
            return ApiResponse(true, message)
        }

        fun error(message: String): ApiResponse {
            return ApiResponse(false, message)
        }
    }
}
